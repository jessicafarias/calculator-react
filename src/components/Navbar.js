import { Link } from 'react-router-dom';

const Navbar = () => (
  <div className="navbar">
    <div className="bg-black">
      <h1> MATH MAGICIAN </h1>
      <h2 className="regular"> CALCULATOR PROJECT </h2>
    </div>
    <div className="bg-black">
      <Link to="/home">Home</Link>
      <Link to="/quote">Quote</Link>
      <Link to="/calculator">Calculator</Link>
    </div>
  </div>
);

export default Navbar;
