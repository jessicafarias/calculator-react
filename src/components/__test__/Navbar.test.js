import { BrowserRouter } from 'react-router-dom';
import React from 'react';
import '@testing-library/jest-dom';
import Navbar from '../Navbar';
import TestRenderer from 'react-test-renderer'; // ES6

describe('Display Navbar', () => {
  it('renders correctly', () => {
    const tree = TestRenderer.create(<BrowserRouter><Navbar /></BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});