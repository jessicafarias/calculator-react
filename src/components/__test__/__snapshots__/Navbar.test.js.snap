// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Navbar renders correctly 1`] = `
<div
  className="bg-image1"
>
  <div
    className="black"
  >
    <div
      className="navbar"
    >
      <div
        className="bg-black"
      >
        <h1>
           MATH MAGICIAN 
        </h1>
        <h2
          className="regular"
        >
           CALCULATOR PROJECT 
        </h2>
      </div>
      <div
        className="bg-black"
      >
        <a
          className="regular"
          href="/home"
          onClick={[Function]}
        >
          Home
        </a>
        <a
          className="regular"
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
        <a
          className="regular"
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </div>
    </div>
  </div>
</div>
`;
